relayer/client.go

```diff
@@ -54,9 +54,13 @@ func (c *Chain) CreateClients(dst *Chain) (modified bool, err error) {
 			AllowUpdateAfterMisbehaviour,
 		)
 
-		// Check if an identical light client already exists
-		clientID, found := FindMatchingClient(c, dst, clientState)
-		if !found {
+		clientID, found := "", false
```

Comment:
```suggestion
var (
    clientID string
    found bool
)
```
You might want to commit these changes yourself since they probably aren't gofmt'd